testing/build.gradle

```diff
@@ -24,6 +26,9 @@ android {
   }
 
```

Comment:
we should add this here.
```
  buildTypes {
    debug {
      testCoverageEnabled true
    }
  }
```

Reply:
test coverage is enabled here.  I got the correct report here